rfid2/rfid2.ino

```diff
@@ -0,0 +1,85 @@
+#include <Keyboard.h>
+#include <SPI.h>
+#include <MFRC522.h>
+
+#define SS_PIN 10
+#define RST_PIN 5
+#define KEY_RETURN 0xB0  // The hex value for the return key is 0xB0.
+
+MFRC522 mfrc522(SS_PIN, RST_PIN);
+char Enter = KEY_RETURN;  // Return key is declared as Enter.
+String readid;
+const String cards[] = {
+  "3A3D9D9E",
+  "3AF3BE9E",
+  "E222CF1D"
+  };  // Change this value to the UID of your card.
+int numberOfCards = (sizeof(cards) / sizeof(cards[0]));
+bool screenLocked = false;
+
+void setup() {
+  Serial.begin(9600);
+  Keyboard.begin();
+  SPI.begin();
+  mfrc522.PCD_Init();
+  Serial.println("Setup complete. Scan your RFID tag.");
+}
+
+void temp(byte *buffer, byte bufferSize) {
+  // Function to store card UID as a string datatype.
```

Comment:
This function does not store anything it just prints to the serial.

Reply:
Thanks for pointing out .

Reply:
Again where are the changes on this branch?